modules/post-install-edge-aws-extend-machineset.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/aws-compute-edge-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="post-install-edge-aws-extend-machineset"]
+= Creating a machine set manifest for AWS Local Zones node
+
+After you created subnets in an AWS Local Zones, you can create a machine set manifest.
+
+The installation program sets the following labels for the `edge` machine pools at cluster installtion time:
+
+* `machine.openshift.io/parent-zone-name: <value_of_ParentZoneName>`
+* `machine.openshift.io/zone-group: <value_of_ZoneGroup>`
+* `machine.openshift.io/zone-type: <value_of_ZoneType>`
+
+The following example in the procedure details how you can create a machine set configuraton that matches the `edge` compute pool configuration.
+
+.Procedure
+
+* To preserve those labels when creating the machine set manifest manually, you must gather the AWS API by running the following command:
```

Comment:
```suggestion
* Manually preserve`edge` machine pool labels when creating the machine set manifest by gathering the AWS API and running the following command:
```

Reply:
I suggest this, just to make the procedure step more actionable from the beginning and clarify "those" labels.